lib/evaluate/formatting.cc

```diff
@@ -56,7 +56,7 @@ std::ostream &ConstantBase<RESULT, VALUE>::AsFortran(std::ostream &o) const {
         Result::category == TypeCategory::Complex) {
       value.AsFortran(o, Result::kind);
     } else if constexpr (Result::category == TypeCategory::Character) {
-      o << Result::kind << '_' << parser::QuoteCharacterLiteral(value);
+      o << Result::kind << '_' << parser::QuoteCharacterLiteral(value, false);
```

Comment:
Should line 95 `o << parser::QuoteCharacterLiteral(value);` also be changed to not always insert backslash escape or is this left on purpose ?

Reply:
I don't think that it matters here, but I'll change it for consistency.